examples/lighting-app/silabs/src/ColorFormat.cpp

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
should probably be part of the RGBLedWidget. The implementation is also C centric. No class, no namespace or struct

Reply:
Added `ColorConverter` class and added methods for conversion.